### PR TITLE
Comments out all those nasty anti-stuns like we had pre-rebase.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
We removed these pre-rebase, no reason not to do so now. 

:cl: Guyonbroadway
del: Removed the recipes for Meth and Ephedrine
del: Removed the ling adrenals evolution
del: Removed Adrenal implants from the traitor uplink
del: Removed Stimpacks from the traitor
del: Removed adrenal implants from the R&D protolathe
tweak: Unholy water no longer reduces stun times
tweak: Bath salts now always comes with a chance you will drop your held items as opposed to while overdosing
tweak: nicotine also no longer reduces stun times, it still keeps it's stamina regen though.
/:cl:

[]: # (Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:)

### CODE REVIEW

**Comment by KayeArray** [comment]
jenkins kill

**Comment by GuyonBroadway** [comment]
I think I see the problem.

**Comment by GuyonBroadway** [comment]
Jenkins Unkill

**Comment by Sarcalogo** [comment]
Good work.

**Comment by trifornt-tinfornt** [comment]
This is something that should have never been returned. Alas, rushed rebase.
Can't wait to see all of the people once again defending them, saying that they're "fine" even though they're a hard counter to the entire shitty combat system that is spam stuns then cuff.
Remove the shitty hugbox items that were designed around a completely different playerbase that has admins that restrict murderboning and powergaming to begin with, then we can remove the crappy balance in stuns - hopefully.

Merge this. They make the entire game about them, which is not good design. Sorry Goofball.

**Comment by KayeArray** [comment]
dem anti stuns are nastay shit boi

**Comment by Chnkr** [comment]
no, this is **bad**

File: code/game/gamemodes/changeling/powers/adrenaline.dm [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (ghost) [inline code/game/gamemodes/changeling/powers/adrenaline.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 2]
```diff
@@ -40,9 +40,9 @@
 	if(prob(1))
 		var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel alert.","You feel rugged.")
 		M << "<span class='notice'>[smoke_message]</span>"
-	M.AdjustParalysis(-1, 0)
-	M.AdjustStunned(-1, 0)
-	M.AdjustWeakened(-1, 0)
+	//M.AdjustParalysis(-1, 0)
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 3]
```diff
@@ -40,9 +40,9 @@
...
+	//M.AdjustStunned(-1, 0)
+	//M.AdjustWeakened(-1, 0)
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 4]
```diff
@@ -40,9 +40,9 @@
 	if(prob(1))
 		var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel alert.","You feel rugged.")
 		M << "<span class='notice'>[smoke_message]</span>"
-	M.AdjustParalysis(-1, 0)
-	M.AdjustStunned(-1, 0)
-	M.AdjustWeakened(-1, 0)
+	//M.AdjustParalysis(-1, 0)
+	//M.AdjustStunned(-1, 0)
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 5]
```diff
@@ -224,9 +224,9 @@
 /datum/reagent/fuel/unholywater/on_mob_life(mob/living/M)
 	if(iscultist(M))
 		M.drowsyness = max(M.drowsyness-5, 0)
-		M.AdjustParalysis(-1, 0)
-		M.AdjustStunned(-2, 0)
-		M.AdjustWeakened(-2, 0)
+		//M.AdjustParalysis(-1, 0)
+		//M.AdjustStunned(-2, 0)
+		//M.AdjustWeakened(-2, 0)
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 6]
```diff
@@ -224,9 +224,9 @@
 /datum/reagent/fuel/unholywater/on_mob_life(mob/living/M)
 	if(iscultist(M))
 		M.drowsyness = max(M.drowsyness-5, 0)
-		M.AdjustParalysis(-1, 0)
-		M.AdjustStunned(-2, 0)
-		M.AdjustWeakened(-2, 0)
+		//M.AdjustParalysis(-1, 0)
+		//M.AdjustStunned(-2, 0)
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 7]
```diff
@@ -224,9 +224,9 @@
 /datum/reagent/fuel/unholywater/on_mob_life(mob/living/M)
 	if(iscultist(M))
 		M.drowsyness = max(M.drowsyness-5, 0)
-		M.AdjustParalysis(-1, 0)
-		M.AdjustStunned(-2, 0)
-		M.AdjustWeakened(-2, 0)
+		//M.AdjustParalysis(-1, 0)
```
Comment (ghost) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/recipes/drugs.dm [hunk 8]
```diff
@@ -19,13 +19,14 @@
 	required_reagents = list("diphenhydramine" = 1, "morphine" = 1, "cleaner" = 1, "potassium" = 1, "phosphorus" = 1, "welding_fuel" = 1)
 	mix_message = "The mixture dries into a pale blue powder."
 	required_temp = 380
-
+/*
```
Comment (ghost) [inline code/modules/reagents/chemistry/recipes/drugs.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 9]
```diff
@@ -120,12 +120,14 @@
 	required_temp = 370
 	mix_message = "The mixture rapidly turns into a dense pink liquid."
 
+/*
```
Comment (ghost) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/research/designs/medical_designs.dm [hunk 10]
```diff
@@ -366,7 +366,7 @@
 	materials = list(MAT_METAL = 800, MAT_GLASS = 500, MAT_GOLD = 500)
 	build_path = /obj/item/weapon/implantcase/freedom
 	category = list("Medical Designs")
-
+/*
```
Comment (ghost) [inline code/modules/research/designs/medical_designs.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/uplink/uplink_item.dm [hunk 11]
```diff
@@ -786,14 +786,15 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/weapon/storage/backpack/satchel/flat
 	cost = 2
 	surplus = 30
-
+/*
```
Comment (ghost) [inline code/modules/uplink/uplink_item.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

File: code/modules/uplink/uplink_item.dm [hunk 12]
```diff
@@ -1103,14 +1104,15 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 4
 	// An empty uplink is kinda useless.
 	surplus = 0
-
+/*
```
Comment (ghost) [inline code/modules/uplink/uplink_item.dm]: **DO NOT COMMENT THINGS OUT.**

**REMOVE IT.**
**OR IMPROVE IT.**

**Review by ghost — CHANGES_REQUESTED** [review]
I personally don't agree with any of this, but the playerbase seems to want it.

I'd personally recommend keeping all the chems, and just making them harder to use, more brain damage, more toxins, chances for a heart attack and the such

File: code/game/gamemodes/changeling/powers/adrenaline.dm [hunk 13]
```diff
@@ -1,22 +1 @@
...
-	user.SetParalysis(0)
-	user.SetStunned(0)
-	user.SetWeakened(0)
-	user.reagents.add_reagent("changelingAdrenaline", 10)
-	user.reagents.add_reagent("changelingAdrenaline2", 2) //For a really quick burst of speed
-	user.adjustStaminaLoss(-75)
-	feedback_add_details("changeling_powers","UNS")
-	return 1
-
+//Nothing to see here
```
Comment (ghost) [inline code/game/gamemodes/changeling/powers/adrenaline.dm:1]: This is still bad, remove the file, remove it from the `.dme`

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
kellyhandicapparking_jpg_630x1200_upscale_q85
+1

**Comment by ghost** [comment]
Actually, you'd be better off making sure each chemical you removed is no longer referenced via chemid in the code. It currently does not fulfill this.

**Comment by ghost** [comment]
https://github.com/HippieStation/HippieStation13/search?utf8=%E2%9C%93&q=ephedrine
https://github.com/HippieStation/HippieStation13/search?utf8=%E2%9C%93&q=methamphetamine
https://github.com/HippieStation/HippieStation13/search?utf8=%E2%9C%93&q=changelingAdrenaline
https://github.com/HippieStation/HippieStation13/search?utf8=%E2%9C%93&q=changelingAdrenaline2

**Comment by PopNotes** [comment]
Please god don't fucking remove ling adrenals

**Comment by PopNotes** [comment]
And _how_ are stimpacks so much of a problem they need to be outright removed?

**Comment by trifornt-tinfornt** [comment]
It's because antistuns are a bad idea to begin with.
Changelings are OP and boring, they don't fit Hippie at all. Sure, they aren't as OP as they were before but they're still really lame. Free antistuns for anyone is a terrible idea.
They need to be removed because they're a hard counter to 90% of combat. It's not like they have a counter themselves, either.
They were removed last rebase and the game was better for it. Now they're back because rushed rebase.
They should be removed again, and nobody will miss them except for the players who used them as a crutch to get out of their poor play.

**Comment by KittyKathy** [comment]
So this is a botany buff? I mean I like botany and all that but giving them the only way to get antistuns as nonantag that isn't crappy bathsalts is pretty bad. This easy solution is not the one because you just removed the recipe for some of the chemicals and only really fixed nicotine and unholy water. 

Omega Weed the plant contains Crank and Meth, Ambrosia deus and Tea astra contains synaptizine, and Coffee robusta contains ephedrine. This also does not account for the fact that strange plants can contain I think any synthetically made chemical as I once got meth trait from one.

There are probably other ways to get these chemicals that tg added like borers that can make meth in their hosts. The best solution you could have done as a first step would probably be to JUST ONLY delete the part of the code where the chemical/implant/ability calls adjuststuns()/adjustweaken()/adjustParalysis() so they still exist as speed boosters and healing chemicals in the case of stimulants(the chemical). You know like what you did with nicotine and unholy water.

The adrenaline implant was pretty useful for the omnizine it gave which could have saved you in light crit and also the ability to increase speed at will.

The only thing the "players" complained about was that it removed stuns so maybe that is the only thing that should be removed from the antistuns.

**Comment by trifornt-tinfornt** [comment]
Drugs that just increased speed with some downsides (like jittery movement/double stun time like Kierany's heroin idea) would be fine. It'd be lame and easily abused in the long run, I could guarantee it. Remember hyperzine?
Botany could probably get some changes in the future and to be honest Botany is pretty OP in their own right.
I disagree with antistuns at their core, everyone hates them except for the people who powergame every round and use them. Even then I've seen some powergamers criticize antistuns. So it's not like the people who hate antistuns are so small that they should be put in quotations.
I do think that traitors should get some kind of edge to keep them going in fights, but that's what all of those trinkets and gadgets are for. If you get stunned, tough shit. That's SS13. Maybe a healing implant with omnizine would be nice for if they didn't finish you off, but that's it.
Other than that, drugs like meth should not make you gods. They should function like Space Drugs and be a joke chem or even a kill chem.

**Comment by Sarcalogo** [comment]
Wew lad. If only the old repo wasn't deleted, I would have screenshot what Kitty Kathy said to me when I did the exact same thing this guy did. Freaked out and told me stuff like "You never listen" "You don't know what you are doing." "You left shit out YOU FUCK EVERYTHING UP!" "LET ME DO IT INSTEAD" 

Comical.

**Comment by Thelolswat** [comment]
The fact that a chemist can get a crap ton of 9u meth patches, and a stunbaton, can control the entire station because there is no way to put them down unless you have godlike aim with a disabler is shitty.

Should keep the antistun part, but add extremely debilitating effects (like random heart attacks, a very long stun once it wears off, taking toxin damage, dropping items, hitting yourself with items).

**Comment by KayeArray** [comment]
While removing them is the easiest thing to do, I am also still of the crowd that believes they can be improved or at least re-purposed rather than completely removed.

**Comment by Sarcalogo** [comment]
Nobody cares you faggot, these need to be removed and it is literally just a port of a feature that was present before the rebase.

**Comment by KayeArray** [comment]
I don't disagree with you there, I just also agree with the sentiment that these drugs could return with a different function as opposed to being gone forever.

**Comment by trifornt-tinfornt** [comment]
There are few times I agree with Sarca.
This is one of them.
I also agree that the drugs could probably be repurposed into something else, personally I think they should just be different versions of Space Drugs - perhaps more intensive so you could use them as floor pill or even smoke to distract and weaken crowds.

**Comment by KayeArray** [comment]
...That is what I'm suggesting

**Comment by ghost** [comment]
I agree with @banthebantz here

**Comment by trifornt-tinfornt** [comment]
Yeah. We should bring them back in the future as forms of Space Drugs or something.
But for now we gotta kill 'em.

**Comment by GuyonBroadway** [comment]
Figured out a better way to nerf the fuck out of anti-stuns, going to make a new PR on it.

**Comment by trifornt-tinfornt** [comment]
Nerfing antistuns instead of removing them.
Welp. Here we go again with the dumb Kierany shit that didn't work.

**Comment by KayeArray** [comment]
JUST MAKE THEM A MEME

**Comment by trifornt-tinfornt** [comment]
HOLY SHIT JUST MAKE THEM A MEME
CTRL+C CTRL+V SPACE DRUGS CODE

**Comment by KayeArray** [comment]
AN EVEN GREATER MEME

**Comment by trifornt-tinfornt** [comment]
A MEME SO GREAT THAT THE SERVER CAN'T HANDLE IT

**Comment by Sarcalogo** [comment]
I change my mind, you did a a terrible job. How many times do we need to try "new" anti-stun systems? The whole concept is fucking busted, how many times do we have to state this. I'll make it simple for you to understand.

\>People complain about anti-stuns
\>Someone tries to remove them
\>A bunch of coders who have never played the game, and idiots who have scarcely played the game a few months come along
\>NO, NO, IT'S REMOVING. WE SHOULD RE-TOOL. STUNS ARE TOO POWERFUL!! THEY NEED A COUNTER!!
\>"Nerf" anti stuns
\>Doesn't do shit because it's still at it's core a FUCKING ANTI-STUN YOU IDIOTS. THAT'S THE PROBLEM. THE PROBLEM IS THAT IT REMOVES STUNS.
\>Idiot coders wipe a bead of sweat off their forehead
\>"Phew... another fine job by me, the problem is solved!'
\>People still go around abusing anti-stuns ruining the entire balance of the game
\>Powergamers find a way to mitigate "nerf" with almost no effort
\>Continue to do so
\>Months later, as the idiots catch on, people complain about anti-stuns
\>The cycle repeats until it is somehow broken

  YEARS AND YEARS AGO I wrote ten page essays about this shit on 4chan's /vg/. Why? Why do people keep making the same mistakes?

**Comment by MethMedic** [comment]
JUST REMOVE THEM. We TRIER nerfing them before and they found a way around it. The only nerf that would be acceptable is an extremely fast metabolic rate, extremely low OD level (5u) and effects that are so bad it's not even worth it. Seizures, brain damage, toxin damage, random overheating, and possible heart attacks. You can't just have one or two of those as they can mix them with chems that stop them. Make it have so many bad side effects that to counter them all, you'd need an insane amount of other chems

**Comment by cacogen** [comment]
I did not read any of the comments here except the last one.

However, ODs and fucking you up is a shitty way to nerf anything. Either make the effect less (e.g. less of a speed boost, less of a stun reduction) or make it last less time. Still preferential to not having it, but not the complete and utter bullshit we have currently.

Personally I've never had any problem with antistuns, but that's probably because I've never had them used against me. I've only ever made omega weed as a botanist and given it all the good shit like ephedrine from coffee robusta and bicaridine, kelotane and omnizine from ambrosia and run around the station pushing people the length of entire hallways in the space of a second.

**Comment by JohnMcLaime** [comment]
Just remove stuns,

**Comment by MethMedic** [comment]
Either nerf them so hard they are pointless or remove them. Make stuns stamina based. Bam it's fixed. Took me 5 seconds.

**Comment by Sarcalogo** [comment]
\>Remove stuns
\>Make stuns stamina based
I can't tell what is stupider. Removing stuns or defending anti-stuns.

**Comment by Swagile** [comment]
we nerfed the shit out of one anti stun

ander found a way to use several chems together to negate the bad effects of an even better anti stun

rinse and repeat until mcbawg put his foot down and completely removed anti stuns

everyone was happy, no meme anti-stuns were added, people moved on

we don't fucking need to nerf anti stuns, we just need to remove them

**Pull request closed without merging**